### ISSUE TITLE
Is there a way to escape the prism face in certain cases

### ISSUE THREAD

**Issue by danielneal** [alphapapa/prism.el#28]
I'd like my docstrings always to be a certain color. 

They are usually given `font-lock-doc-face` but when prism is running, they become `prism-level-1-strings`. 

Is there a way to escape the prism coloring for a specific face?

**Comment by alphapapa** [comment]
No, the best you could do would be to set that face to the docstring face.

**Comment by danielneal** [comment]
Thanks for the tip!

It also comes up when I use a debugger - the debugger highlights the face, but the prism-chosen face wins

I tried to get ChatGPT to help but it was beyond useless ha. 

Would you be against having a think in case any ideas occur to you on how I could make this usecase work? 

Anyway, I'm very happy using prism, it brings me much joy.
